src/main/java/com/google/sps/data/Person.java

```diff
@@ -0,0 +1,62 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/** Represents a user on the site. The email is used as the key to the user table. */
+public class Person {
+  private String email, name, company, job, linkedIn;
+
+  public Person(String email, String name, String company, String job, String linkedIn) {
+    this.email = email;
+    this.name = name;
```

Comment:
name needs to be a first name and a last name

Reply:
Done